rocket_sqlite_oracle/src/db/kraken.rs

```diff
@@ -255,9 +255,23 @@ mod wire {
         BitcoinParseAmount(#[from] ParseAmountError),
     }
 
+    #[derive(Debug, PartialEq, Clone, Copy)]
+    pub enum Units {
+        Millicents,
+    }
```

Comment:
Each variant of this enum only describes a single unit, hence I'd name this singular as well.